### PR TITLE
Adding refresh option for Nearby

### PR DESCRIPTION
**Pull request by nextLane**
Clicking on this recreate the activity.
An improvement over this can be however to just re-calculate the closest locations than to fetch and parse the CSV too. 
#375

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:2.3.0'
```
Comment (maskaravivek) [inline build.gradle]: @nextLane why is this required?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Wed Aug 17 16:02:52 NZST 2016
+#Fri Mar 03 19:41:57 IST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```
Comment (maskaravivek) [inline gradle/wrapper/gradle-wrapper.properties]: is this required?

File: app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java [hunk 3]
```diff
@@ -40,12 +43,32 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                return super.onOptionsItemSelected(item);
+        }
+    }
     @Override
     protected void onResume(){
         super.onResume();
     }
 
+    protected void refreshView()
+    {
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/nearby/NearbyActivity.java]: nit: formatting

**Comment by domdomegg** [comment]
Not original poster, but `build.gradle` and `gradle-wrapper.properties` automatically get updated in Android Studio I think, which is probably the cause of those 2 edits. Probably want to undo them before merge - maybe do in a different PR.

**Comment by misaochan** [comment]
Yeah, the updates are automatic in Android Studio. This could do with some discussion: do we want to update Gradle whenever an update is available? Anyone else, please feel free to chime in.

**Comment by domdomegg** [comment]
I think updating Gradle when updates are available is probably best - but defo should be separate pull req. to make sure it doesn't break anything.

**Comment by misaochan** [comment]
Thanks @domdomegg , I agree. @nextLane , could you please submit the Gradle update in a separate PR? Thanks so much. :)

**Comment by misaochan** [comment]
Could a reviewer please test and merge now that the conflicts are resolved? I was just notified of the new commits and am away from my computer til tomorrow, sorry. :/

**Comment by maskaravivek** [comment]
@misaochan Have tested and merged this PR. We might need to update the icons though. It can be done in another PR.

**Comment by misaochan** [comment]
Thanks @maskaravivek ! Yeah, another PR would be good.
